hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/aws_sdk_upgrade.md

```diff
@@ -66,6 +66,25 @@ The change in interface will mean that custom credential providers will need to
 implement `software.amazon.awssdk.auth.credentials.AwsCredentialsProvider` instead of
 `com.amazonaws.auth.AWSCredentialsProvider`.
 
+[HADOOP-18980](https://issues.apache.org/jira/browse/HADOOP-18980) introduces extended version of
+the credential provider remapping. `fs.s3a.aws.credentials.provider.mapping` can be used to
+list comma-separated key-value pairs of mapped credential providers that are separated by
+equal operator (=). The key can be used by `fs.s3a.aws.credentials.provider` config, and it
+will be translated into the specified value of credential provider class based on the key-value
+pair provided by this config.
+
+For example, if `fs.s3a.aws.credentials.provider.mapping` is set with value:
+
```

Comment:
Just tried xml snippet to see how it looks and actually it looks better because now the example is much clear. Please let me know what you think of the latest revision.